pkg/volume/csi/csi_plugin.go

```diff
@@ -77,24 +80,51 @@ type csiDriversStore struct {
 	sync.RWMutex
 }
 
+// RegistrationHandler is the struct that handles plugins
 // TODO (verult) consider using a struct instead of global variables
+// RegistrationHandler is the handler which is fed to the pluginwatcher API.
```

Comment:
This looks good from the CSI side.  There are some other housekeeping stuff that needs to be done and it can wait after this is merged.